tst/EnergyPlus/unit/ReportSizingManager.unit.cc

```diff
@@ -353,3 +353,25 @@ TEST_F( EnergyPlusFixture, ReportSizingManager_RequestSizingZone ) {
 	ZoneSizingInput.deallocate();
 
 }
+
+TEST_F( EnergyPlusFixture, ReportSizingManager_ReportSizingOutputTest ) {
+	// issue #6112
+	std::string CompName;
+	std::string CompType;
+	std::string VarDesc;
+	std::string UsrDesc;
+	Real64 VarValue;
+	Real64 UsrValue;
+
+	// input values
+	CompType = "BOILER:HOTWATER";
+	CompName = "RESIDENTIAL BOILER ELECTRIC";
+	VarDesc = "Design Size Nominal Capacity[W]";
+	VarValue = 105977.98934;
+	UsrDesc = "User-Specified Nominal Capacity[W]";
+	UsrValue = 26352.97405;
+
+	// boiler hot water autosizing and userspecified nominal capacity reporting
+	ReportSizingManager::ReportSizingOutput( CompType, CompName, VarDesc, VarValue, UsrDesc, UsrValue );
```

Comment:
My intent here was to add to the unit test in reportsizingmanager.unit.  Would it work to combine the last half of what you added to sqlite.unit with what you have here?  As it is now, neither test is really checking to be sure that reportsizingmanager wrote both sets of information to the sql output.

Also, please add a description of this change to \src\Transition\OutputRulesFiles\OutputChanges8-7-0-to-8-7-0.md  This will be a new file.  You can use the 8-6-0-to-8-7-0 as a starting point.